docs/extensions/guides/README.md

```diff
@@ -24,5 +24,9 @@ Each guide or sample will include:
 
 | Sample | APIs |
 | ----- | ----- |
+[custom-resource-page](https://github.com/lensapp/lens-extension-samples/tree/master/custom-resource-page) | LensRendererExtension <br> K8sApi.KubeApi <br> K8sApi.KubeObjectStore <br> Component.KubeObjectListLayout <br> Component.KubeObjectDetailsProps <br> Component.IconProps |
```

Comment:
Should "helloworld" be first?

Reply:
Probably? It's just alphabetical for now. I figure wait till the list is complete

Reply:
moved it while resolving merge conflict